tournament.json

```diff
@@ -0,0 +1,88 @@
+{
+  "#": [
+    "The inputs and outputs are represented as arrays of strings to improve readability in this JSON file.",
+    "Your track may choose whether to present the input as a single string (concatenating all the lines) or as the list.",
+    "In most cases, it seems to make sense to expect the output as a single string."
+  ],
+  "valid_inputs": {
+    "cases": [
+      {
+        "description": "typical input",
+        "input": [
+          "Allegoric Alaskans;Blithering Badgers;win",
+          "Devastating Donkeys;Courageous Californians;draw",
+          "Devastating Donkeys;Allegoric Alaskans;win",
+          "Courageous Californians;Blithering Badgers;loss",
+          "Blithering Badgers;Devastating Donkeys;loss",
+          "Allegoric Alaskans;Courageous Californians;win"
+        ],
+        "expected": [
+          "Team                           | MP |  W |  D |  L |  P",
+          "Devastating Donkeys            |  3 |  2 |  1 |  0 |  7",
+          "Allegoric Alaskans             |  3 |  2 |  0 |  1 |  6",
+          "Blithering Badgers             |  3 |  1 |  0 |  2 |  3",
+          "Courageous Californians        |  3 |  0 |  1 |  2 |  1"
+        ]
+      },
+      {
+        "description": "incomplete competition (not all pairs have played)",
```

Comment:
Wasn't there some restriction on this problem that only complete tournaments could be tallied? I remember seeing that, but now I can't find it.